spec/features/users/user_can_search_rdv_on_rdv_mairie_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+describe "User can search rdv on rdv mairie" do
+  def json_response
+    JSON.parse(page.html)
+  end
+
+  before do
+    default_url_options[:host] = "http://www.rdv-mairie-test.localhost"
+  end
+
+  around do |example|
+    previous_auth_token = ENV["ANTS_API_AUTH_TOKEN"]
+    ENV["ANTS_API_AUTH_TOKEN"] = ""
+
+    example.run
+
+    ENV["ANTS_API_AUTH_TOKEN"] = previous_auth_token
+  end
+
+  before do
+    travel_to(Time.zone.now)
+    create(:plage_ouverture, :daily, first_day: Time.zone.today, motifs: [motif], lieu: lieu, organisation: organisation, start_time: Tod::TimeOfDay(9), end_time: Tod::TimeOfDay(10))
+  end
+
+  describe "default" do
+    let!(:territory92) { create(:territory, departement_number: "92") }
+    let!(:organisation) { create(:organisation, :with_contact, territory: territory92, verticale: :rdv_mairie) }
+    let(:service) { create(:service) }
+    let!(:motif) { create(:motif, name: "Vaccination", bookable_publicly: true, organisation: organisation, restriction_for_rdv: nil, service: service) }
```

Comment:
Comme les seeds, c'est plus facile à comprendre si on a des exemples du métier ;)
```suggestion
    let!(:motif) { create(:motif, name: "Passeport", bookable_publicly: true, organisation: organisation, restriction_for_rdv: nil, service: service) }
```

Reply:
Fait dans 257133f703d2534a7d3f45e8773e21f5617ba3d4